model/src/main/kotlin/OrtResult.kt

```diff
@@ -326,12 +311,21 @@ data class OrtResult(
      * Return true if the project or package with the given identifier is excluded.
      */
     @Suppress("UNUSED") // This is intended to be mostly used via scripting.
-    fun isExcluded(id: Identifier) =
-        analyzer?.result?.let { repository.config.excludes?.isExcluded(id, this) } == true
+    fun isExcluded(id: Identifier): Boolean
+        = getExcludes().isExcluded(id, this)
```

Comment:
This would fit in a single line now. At least `=` should be in the line above.

Reply:
Done.